specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json

```diff
@@ -886,6 +929,10 @@
         }
       },
       "type": "object",
+      "required": [
+        "codecs",
+        "formats"
+      ],
       "description": "Describes all the settings to be used when encoding the input video with the Standard Encoder."
```

Comment:
These properties have been required since the introduction of this version of the API. The API calls were failing if these properties weren't provided. This change just moves the validation to the client. Failure to provide these properties